org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -36,16 +36,16 @@
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
             <unit id="org.eclipse.xtend.sdk.feature.group" version="0.0.0"/>
             <unit id="org.eclipse.xtext.sdk.feature.group" version="0.0.0"/>
-            <repository location="http://download.eclipse.org/releases/2019-06/"/>
+            <repository location="http://download.eclipse.org/releases/2019-09/"/>
         </location>
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
             <unit id="org.jboss.tools.maven.apt.core" version="0.0.0"/>
             <repository location="https://download.jboss.org/jbosstools/updates/m2e-extensions/m2e-apt/1.5.3-2019-11-08_11-04-22-H22/"/>
         </location>
-	    <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-		    <repository location="http://services.typefox.io/open-source/jenkins/job/lsp4j/job/master/lastSuccessfulBuild/artifact/build/p2-repository/"/>
-		    <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.0.0"/>
-	    </location>
+        <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+           <repository location="https://github.com/testforstephen/lsp4j/raw/p2/p2/"/>
```

Comment:
Can't merge a PR pointing toward a personal build. We need lsp4j PRs to be merged and at the very least points to a snapshot (bad) or release (good) build

Reply:
The upstream lsp4j PRs are merged. And i also replaced it with the upstream lsp4j snapshot url.